web-services/client/src/main/java/datawave/webservice/model/ModelValidator.java

```diff
@@ -0,0 +1,119 @@
+package datawave.webservice.model;
+
+import datawave.query.util.MetadataHelper;
+import datawave.util.UniversalSet;
+import org.apache.accumulo.core.client.TableNotFoundException;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.Map;
+import java.util.Set;
+
+public class ModelValidator {
```

Comment:
static utility should have a private constructor to enforce exclusive access via static methods